rice-namenode/include/ClientNamenodeProtocolImpl.h

```diff
@@ -51,6 +52,10 @@ class ClientNamenodeTranslator {
 		std::string setPermission(std::string);
 		std::string recoverLease(std::string);
 		std::string concat(std::string);
+		std::string getEZForPath(std::string input);
+		std::string setOwner(std::string input);
+		std::string getContentSummary(std::string input);
```

Comment:
looks like we support this one, let's move it up in the header so it's not under the section "not supported"